packages/kbn-docs-utils/src/api_docs/build_api_declarations/buid_api_declaration.test.ts

```diff
@@ -129,7 +129,7 @@ it('Test ReactElement signature', () => {
   // There is a terrible hack to achieve this, but without it, ReactElement<Props> expands to include the second default generic type
   // (ReactElement<Props, string | (any) crazy code here with lots of anys that comes from react types >) and
   // it looks awful.
-  expect(def.signature![2]).toBe('>');
+  expect(def.signature![2]).toBe(', string | React.JSXElementConstructor<any>>');
```

Comment:
@stacey-gammon I'm not 100% sure which type we updated to cause this failure, but I've updated the tests even though it seems this isn't what we want. Can you point me in the right direction to fix this correctly?

Reply:
It's this line: https://github.com/elastic/kibana/blob/main/packages/kbn-docs-utils/src/api_docs/build_api_declarations/get_signature.ts#L93. The string to match probably needs to get changed. If you want to skip the test and assign it to me I can take a look.